roll.py

```diff
@@ -408,6 +444,17 @@ def main():
         elif lib.args.command == "clean-aa":
             account_abstraction.clean()
 
+        if lib.args.command == "remote":
+            ansible.generate_inventory(config)
+            log_file = open("logs/ansible.log", "w")
+            ansible.run_playbook(
+                config,
+                "setup and start remote deployment",
+                log_file=log_file,
+                playbook="rollop.yml",
+                tags=["setup", "l1", "l2", "restart"]
+            )
```

Comment:
Down the line we'll want to have `--remote` flag that we can pass to most command, so we're able to deploy the whole system remotely, or just the l1, l2, or l2 components separately.